packages/cardpay-cli/claim-settlement/execute.ts

```diff
@@ -32,10 +32,14 @@ export default {
     let claimSettlementModule = await getSDK('ClaimSettlementModule', ethersProvider, signer);
     let blockExplorer = await getConstant('blockExplorer', ethersProvider);
     let onTxnHash = (txnHash: string) => console.log(`Transaction hash: ${blockExplorer}/tx/${txnHash}`);
+    let claim = await claimSettlementModule.defaultStakingClaim(moduleAddress, payeeSafeAddress);
```

Comment:
This is still constructing & signing the claim - if we remove this part and the defaultstakingclaim function we'd have a distinct sign & claim cli function that seems good.